src/content/api/cli.md

```diff
@@ -447,3 +447,11 @@ NODE_OPTIONS="--max-old-space-size=4096 -r /path/to/preload/file.js" webpack
 | `0`       | Success                                            |
 | `1`       | Errors from webpack                                |
 | `2`       | Configuration/options problem or an internal error |
+
+## Environment Variables
+
+| Environment Variable | Description                                  |
+| -------------------- | -------------------------------------------- |
+| `WEBPACK_SERVE`      | `true` if `serve\|s` command is being used.  |
+| `WEBPACK_BUILD`      | `true` if `build\|bundle\|b` is being used.  |
+| `WEBPACK_WATCH`      | `true` if `--watch\|watch\|w` is being used. |
```

Comment:
Let's provide simple example with `webpack.config.js`

Reply:
We need clarify here that `WEBPACK_SERVE`/`WEBPACK_BUILD`/`WEBPACK_WATCH` can be used only in `webpack.config.js`, some developer can think it can be used in bundle core too

Reply:
updated